scene/main/viewport.cpp

```diff
@@ -1475,6 +1475,8 @@ void Viewport::_gui_show_tooltip() {
 		gui.tooltip_label->set_text(gui.tooltip_text);
 		base_tooltip = gui.tooltip_label;
 		panel->connect(SceneStringName(mouse_entered), callable_mp(this, &Viewport::_gui_cancel_tooltip));
+	} else {
+		return;
 	}
```

Comment:
This contradicts the comment above:
```
// Controls can implement `make_custom_tooltip` to provide their own tooltip.
// This should be a Control node which will be added as child to a TooltipPanel.
Control *base_tooltip = tooltip_owner->make_custom_tooltip(gui.tooltip_text);
```

And will break functionality of `make_custom_tooltip` (which is exposed to scripting).